qa/rpc-tests/cfund-paymentrequest-payout.py

```diff
@@ -169,36 +167,30 @@ def run_test(self):
         for x in range(self.num_nodes):
             assert(self.nodes[x].getbestblockhash() == bestBlockHash)
             assert(self.nodes[x].getpaymentrequest(paymentReq)["state"] == 1)
-            assert(self.nodes[x].getpaymentrequest(paymentReq)["paidOnBlock"] == "0000000000000000000000000000000000000000000000000000000000000000")
+            assert(self.nodes[x].getpaymentrequest(paymentReq)["stateChangedOnBlock"] != "0000000000000000000000000000000000000000000000000000000000000000")
```

Comment:
why has this assertion been inverted?

Reply:
paidOnBlock does not exist anymore, but stateChangedOnBlock is used combined with state==6 to show in which block a payment request has been paid. line 172 checks the payment request is not null